.vscode/launch.json

```diff
@@ -0,0 +1,13 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        // Run this in your terminal at the same time:
```

Comment:
@khang859 Github is doing some crazy highlighting here, but worth noting that VS Code dgaf about these comments